kiva/basecore2d.py

```diff
@@ -639,9 +640,37 @@ def arc(self, x, y, radius, start_angle, end_angle, cw=False):
         self.state.current_point = pts[-1]
 
     def arc_to(self, x1, y1, x2, y2, radius):
-        """
-        """
-        raise NotImplementedError("arc_to is not implemented")
+        """ Draw a circular arc from current point to tangent line
+
+        The arc is tangent to the line from the current point to ``(x1, y1)``,
+        and it is also tangent to the line from ``(x1, y1)`` to ``(x2, y2)``.
+        ``(x1, y1)`` is the imaginary intersection point of the two lines
+        tangent to the arc at the current point and at ``(x2, y2)``.
+
+        If the tangent point on the line from the current point to ``(x1, y1)``
```

Comment:
What's the behaviour when all three points (i.e., the current point, `(x1, y1)` and `(x2, y2)`) are collinear? Do we simply get a line from the current point to `(x1, y1)`? Is this corner case worth documenting / testing?